src/app/plant-metadata/plant-metadata.component.html

```diff
@@ -47,9 +47,10 @@
   <button
     mat-stroked-button
     style="margin: 10px;"
-    (click)="onSubmit()"
+    type="submit"
```

Comment:
We were calling `onSubmit()` twice. This prevents and will use forms `onSubmit()`